src/main/java/core/mate/academy/service/impl/BulldozerProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.service.impl;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private final List<Bulldozer> list = new ArrayList<>();
```

Comment:
what will the method return to you on the second call: 2 elements or 4? likewise in the following cases